assets/src/components/notificationDrawer.tsx

```diff
@@ -201,7 +201,12 @@ const NotificationCard = ({
   return (
     <button
       className={`m-notification-drawer__card m-notification-drawer__card--${notification.state}`}
-      onClick={() => openVPPForCurrentVehicle(notification)}
+      onClick={() => {
+        if (window && window.FS) {
+          window.FS.event("Chelsea bridge notification clicked")
```

Comment:
Won't this trigger when any notification is clicked, not just Chelsea St Bridge notifications?